plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb

```diff
@@ -128,15 +96,20 @@ def set_docker_image
       container.image = docker_path
     end
 
+    # helpful env vars, also useful for log tagging
     def set_env
       env = (container.env || [])
 
       # static data
       metadata = release_doc_metadata
-      [:REVISION, :TAG, :PROJECT, :ROLE, :DEPLOY_ID, :DEPLOY_GROUP].each do |k|
+      [:REVISION, :TAG, :DEPLOY_ID, :DEPLOY_GROUP].each do |k|
```

Comment:
Not sure I follow, but does means that the rev, tag, etc. will be added as labels, and the yaml file should have the project and roles labels in the template for the pods that will be created?
Not that I have a problem with that, just want to be sure I understand (sorry, I'm a newby in web and ruby :))